web/client/epics/featuregrid.js

```diff
@@ -424,9 +425,21 @@ module.exports = {
             return Rx.Observable.of(setControlProperty("drawer", "enabled", false), toggleTool("featureCloseConfirm", false));
         }),
     onOpenAdvancedSearch: (action$, store) =>
-        action$.ofType(OPEN_ADVANCED_SEARCH).switchMap(() =>
-            Rx.Observable.of(
+        action$.ofType(OPEN_ADVANCED_SEARCH).switchMap(() => {
```

Comment:
move into query panel. Check if you can intercept better when the query panel is shown.